node/pkg/driver/node_utils.go

```diff
@@ -239,6 +243,26 @@ func (n NodeUtils) ParseFCPorts() ([]string, error) {
 	return fcPorts, nil
 }
 
+func (n NodeUtils) IsFCExists() bool {
+	return n.IsPathExists(FCPath) && !n.isEmptyDir(FCPath)
+}
+
+func (n NodeUtils) isEmptyDir(path string) bool {
+	f, _ := os.Open(path)
+	defer f.Close()
+
+	_, err := f.Readdir(minFilesInNonEmptyDir)
+
+	if err != nil {
+		if err != io.EOF {
+			logger.Warningf("Check is directory %s empty returned error %s", path, err.Error())
```

Comment:
Typo, but anyway, I would change the message to:
```suggestion
			logger.Warningf("Failed to read content of directory %s. Error %s", path, err.Error())
```

Reply:
I wanted to do like [here](https://github.com/IBM/ibm-block-csi-driver/blob/259b3d40a7d44c476d7d232631c5646c6aebaf76/node/pkg/driver/node_utils.go#L246) but I agree that what you wrote is better, do you still want me to change it? or keep it like this that it will be similar to the other `logger.Warningf`

Reply:
We'll change it in a different PR. I created a new ticket for that: CSI-3149